pyfstat/mcmc_based_searches.py

```diff
@@ -1233,37 +1233,8 @@ def plot_cumulative_max(self, **kwargs):
 
         if hasattr(self, "search") is False:
             self._initiate_search_object()
-        if self.binary is False:
```

Comment:
I can't put comments on unchanged lines, so I'll put it here: do we still need the `self.generate_loudest()` a few lines above? I don't think the core functions know how to ingest that file any more now,

Reply:
I think this one still applies under the latest diff, unless github is hiding more from me...

Reply:
>I think this one still applies under the latest diff, unless github is hiding more from me...

Before there was an ugly (I know, that's opinionated, but so is `black` and we are happy with it) `if: else` block, the only thing it made wast to pass binary parameters to the function (or not); however, that was not necessary as the function was able to process binary parameters as `None`. Just a clarification.

Reply:
> I can't put comments on unchanged lines, so I'll put it here: do we still need the self.generate_loudest() a few lines above? I don't think the core functions know how to ingest that file any more now,

I agree. Let's kill it and push forward #191 at some point so we are still able to do it but in a more Pythonic way.